tables/flavor.py

```diff
@@ -137,7 +145,7 @@ def flavor_to_flavor(array, src_flavor, dst_flavor):
         return array
 
 
-def internal_to_flavor(array, dst_flavor):
+def internal_to_flavor(array: np.ndarray, dst_flavor: FlavorType) -> Union[Any, list[Any], np.ndarray]:
```

Comment:
The first argument should be `ArrayLike` IMHO

Reply:
Changed to `ArrayLike`